src/compile/frontends/native_compilers.ts

```diff
@@ -116,19 +118,16 @@ export async function httpsGetSync(url: string): Promise<Buffer> {
             res.on("error", (e) => reject(e));
         });
     });
-
-    return await reqPromise;
 }
 
 async function getCompilerMDForPlatform(prefix: string): Promise<CompilerPlatformMetadata> {
     const cachedListPath = path.join(CACHE_DIR, prefix, "list.json");
 
     if (fse.existsSync(cachedListPath)) {
-        //console.error(`Hit cache for list ${prefix}`);
         return fse.readJSONSync(cachedListPath) as CompilerPlatformMetadata;
```

Comment:
We may need to have a logic for recheking, as the list is updated after some time.